### PR TITLE
Portfolio: desktop version: day 2

### PR DESCRIPTION
**Pull request by SSEKPIUS**
Use media queries in CSS to make sure that my website displays correctly on desktop screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,9 +173,6 @@ <h3>Multi-Post Stories</h3>
             <div class="about-top">
```
Comment (muneebulrehman) [inline index.html:158]: The background image for the about section should be aligned to the right of the page and no space between the background image and the right edge. However, in your case, there is some margin to the right of the background image. Kindly fix this issue. Here is a comparison with the Figma design.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

Comment (muneebulrehman) [reply index.html:158]: In the mobile version, the background image is not visible. Kindly fix this issue.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -173,9 +173,6 @@ <h3>Multi-Post Stories</h3>
...
               <h2 class="primary-text">About me</h2>
               <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-              <span class="action">
-                <span>Get My Resume</span>
-              </span>
               <input type="submit" value="Get My Resume">
             </div>
             <div class="about-bottom"></div>
```
Comment (muneebulrehman) [inline index.html:163]: The layout for this section is not as per the Figma template of your choosing. Kindly fix the issues before submitting for a re-review. Also, kindly use the same spacing, margins, padding, and font-size as in the Figma template. 
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @SSEKPIUS , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Usage of semantic HTML

## Required Changes ♻️

- [ ] Kindly update your README file as per this project. Some sections are not relevant to this project or have not been updated yet according to this project. Sections like Getting started, Run tests, and Deployment are not relevant to this project. Sections like Prerequisites and Setup should be updated as per this project. Under the Authors section, there are two authors and both the authors lead to the same person. Kindly remove the second author.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly use the same Font-size, font color, background-color, margins, and padding as in the Figma template that you have chosen. I am marking this as optional because at this point it would be difficult to adjust the project to these specifications. But kindly keep this in mind while working on this project in future.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @SSEKPIUS 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the required changes, but you still have some issues: 

[ ] kindly fix the headline section in the mobile version, the text is unreadable 🤓.
Your design : 

mobile

The Figma : 

desmp

 
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Santiago220991 — APPROVED** [review]
Hi @SSEKPIUS,
I'm Santiago your reviewer for this PR. :wave: :wave:  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: Applied changes required by the last reviewer.
:heavy_check_mark: No linter errors.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**